### ISSUE TITLE
The Members List is maintained as a text file instead of MarkDown file

### ISSUE THREAD

**Issue by Git-Harshit** [codezoned/join_codezoned#51]
The [Members-List](https://github.com/codezoned/join_codezoned/blob/master/Members-List.txt) file in master branch is currently maintained as a standard plaintext file, which is visually lesser appealing as compared to the GitHub standard, MarkDown files (*.md). 

I'd suggest using Markdown files instead, as they have better *formatting* and *text maintenance* options as compared to plaintext (*.txt) files. If community agrees, I won't mind renaming the file, and open another PR for the same. Please mention about your views 👀 on this. 

Pre-Thanks for the same! 😃

**Comment by manisaiprasad** [comment]
Agreed!
Better formatting

**Comment by ionicc** [comment]
Hey @manisaiprasad, This sounds like a good idea. 

Initially we made join_codezoned so as to teach people how to make simplest of PRs and that is, Editing a .txt file 😄 
Do you think changing this to .md files could bring in an aspect of confusion to people who don't understand how Markdown files work? And if it does, How about you edit the Readme to teach people this as well?

Also, We had also planned for this .txt files to become a data api of sorts to project data on codezoned.com.

**Comment by PremApk** [comment]
@ionicc I agree with your point. Instead, we can post the steps in a plain text file and we can create md for the members list. So that people can get to know about Markdown Files. 🙂
